modules/osdk-how-csv-gen-works.adoc

```diff
@@ -5,51 +5,10 @@
 [id="osdk-how-csv-gen-works_{context}"]
 = How CSV generation works
 
-The `manifests/` directory of an Operator project is the standard location for all manifests required to deploy an Operator on OLM. The cluster service version (CSV) generator, called by the `generate bundle` subcommand, requires certain input manifests to construct a CSV manifest; all inputs are read when the command is invoked, along with a CSV base, to idempotently regenerate a CSV.
+Operator bundle manifests, which include cluster service versions (CSVs), describe how to display, create, and manage an application on Operator Lifecycle Manager (OLM). The CSV generator in the Operator SDK, called by the `generate bundle` subcommand, is the first step towards publishing your Operator to a catalog and deploying it with OLM. The subcommand requires certain input manifests to construct a CSV manifest; all inputs are read when the command is invoked, along with a CSV base, to idempotently generate or regenerate a CSV.
```

Comment:
_manage an application on_ OLM. Is this the right wording, _on_? I see it used in a few places; but sounds weird.

Reply:
I'll do: s/on/with/